ui/desktop/electron-app/package.json

```diff
@@ -12,11 +12,10 @@
     "build:config": "node -e \"require('./config/desktop.js').setup()\"",
     "start": "electron-forge start",
     "package": "electron-forge package",
-    "premake": "yarn build:cli && yarn build:config",
-    "make": "electron-forge make",
+    "configure": "yarn build:cli && yarn build:config",
```

Comment:
I think this is fine but this could also just go under the other 2 `build` commands since it just calls both, it could just be `build:all`

Reply:
💯  that makes more sense, I will rename and move it up